### ISSUE TITLE
NTR: Pulmonary hyperplasia (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8995]
New term request Pulmonary hyperplasia (elthat_100204201026): Lung  UMLS: 3686578

**Comment by pnrobinson** [comment]
@MickeySegal
this is not a commonly used term -- what is the context?
if this: https://pubmed.ncbi.nlm.nih.gov/22767559/
then it would be this term: Congenital pulmonary airway malformation HP:0010959
otherwise please provide a definition.

**Comment by MickeySegal** [comment]
That citation refers to "bronchial atresia".

Our context is 
https://omim.org/entry/219000
"Reports of 3 additional cases of pulmonary hyperplasia in the Fraser syndrome were found. In all of these cases the likely mechanism for pulmonary hyperplasia was retention of fetal lung fluid by laryngeal or tracheal obstruction."

**Comment by pnrobinson** [comment]
@MickeySegal 
this does not sound like true hyperplasia; rather it is something like enlargement of the fetal lungs due to fluid retention.
@tsparks -- thoughts?

**Comment by MickeySegal** [comment]
https://rarediseases.info.nih.gov/diseases/6465/fraser-syndrome lists for Fraser syndrome "Pulmonary hypoplasia ... Hypoplastic lung; Hypoplastic lungs; Lung hypoplasia; Poorly developed lungs; Small lung; Underdeveloped lung" and https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5394498/ lists "echogenic lungs... hypoplastic lungs"

But https://pubmed.ncbi.nlm.nih.gov/7747754/ says "We report on 2 sibs with the Fraser cryptophthalmos syndrome who had pulmonary hyperplasia and laryngeal stenosis. A third unrelated patient with Fraser syndrome had laryngeal stenosis, renal agenesis, and normal lung development, rather than the expected pulmonary hypoplasia. Three additional cases of pulmonary hyperplasia in the Fraser syndrome were ascertained from a review. In all of these cases the likely mechanism for pulmonary hyperplasia is retention of fetal lung fluid by laryngeal or tracheal obstruction."

**Comment by pnrobinson** [comment]
@MickeySegal not an expert, but this sounds more like "retention of fetal lung fluid " than true "pulmonary hyperplasia". I cannot access the cited article to look at the evidence...

**Comment by MickeySegal** [comment]
Another article using the term: https://academic.oup.com/cardiovascres/article/78/3/409/269490

**Comment by pnrobinson** [comment]
@mgriese could you help us with this term? Does true Pulmonary hyperplasia exist or is this "retention of fetal lung fluid"?

**Comment by mgriese** [comment]
Pulmonary Hyperplasia is an absolute increase in the number of alveoli, which are mildly enlarged. Pulmonary hyperplasia has been also referred to as polyalveolar lobe and type 3 cystic adenomatoid malformation. The pathogenesis and clinical characteristics of this entity are related to congenital lobar overinflation/emphysema, but should be distinguished. In the latter condition, the volume increase is due to alveolar enlargement as opposed to an absolute increase in alveoli. Radial alveolar counts can be helpful for distinguishing between these two abnormalities.
Pulmonary hyperplasia may be associated with in utero death due to compression of venous return to the heart and/or esophageal compression, resulting in polyhydramnios and fetal hydrops. The therapy for infants with lobar distension causing severe mediastinal shift and normal lung compression is surgical removal of the affected lobe or segment.


Zander, Dani S., and Carol F. Farver. Pulmonary Pathology E-Book : A Volume in Foundations in Diagnostic Pathology Series, Elsevier, 2017

**Comment by mgriese** [comment]
To answer the question: retention of lung fluid generated by the liquid filled airspaces of the lungs themselves from an airway stenosis (or more pronounced atresia/complete clusre results in pulmonary hyperplasia.

**Comment by pnrobinson** [comment]
@mgriese Thanks!
@MickeySegal 

Adding new term: 
- ID: HP:6001258
- label: Pulmonary hyperplasia
- parents: [Abnormal lung morphology (HP:0002088)](https://hpo.jax.org/app/browse/term/HP:0002088)
- pmid: PMID:7747754
- def: Pulmonary Hyperplasia is an absolute increase in the number of alveoli, which are mildly enlarged. Pulmonary hyperplasia has been also referred to as polyalveolar lobe and type 3 cystic adenomatoid malformation.
- comment: The pathogenesis and clinical characteristics of pulmonary hyperplasia are related to congenital lobar overinflation/emphysema, but should be distinguished. In the latter condition, the volume increase is due to alveolar enlargement as opposed to an absolute increase in alveoli. Radial alveolar counts can be helpful for distinguishing between these two abnormalities. Pulmonary hyperplasia may be associated with in utero death due to compression of venous return to the heart and/or esophageal compression, resulting in polyhydramnios and fetal hydrops. The therapy for infants with lobar distension causing severe mediastinal shift and normal lung compression is surgical removal of the affected lobe or segment.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8995
- synonyms:
